### ISSUE TITLE
Provide CLI commands to get volume details from backups

### ISSUE THREAD

**Issue by pradeepkchaturvedi** [vmware-tanzu/velero#4166]
Provide easy way to list PV details from Backup CR and Restore CR. User want to get details from backup and restore operations like list of PVs included, PV size, what is PV backup method(Restic or provider snapshot), time taken for PV backup or restore

**Comment by yanji09** [comment]
@pradeepkchaturvedi is the "--details" in describe backup is enough? the PV information and backup method should be there. Does user care the size of the PV or the size of the backed up data?

**Comment by pradeepkchaturvedi** [comment]
@yanji09 Yes, -- details provide info on PV and PVC. If user wants to know only about PV & PVC they still need to run and browse full details. I was thinking if there can be another filter to the backup describe like "velero backup describe nginx-backup --details --pv" or "velero backup describe nginx-backup --details --pvc".

I noticed in velero backup describe --details command lists both PV & PVC in Resource List section. We may add their size as well. In Persistent Volumes section, PV name mentioned however on Restic Backups section PVC name mentioned. I assume, this is by design.

I think, for provider snapshot PV size would matter however for Restic Backups size of the backed up data would matter. If Velero can also show link between PVC -> PV, that would be great.

**Comment by eleanor-millman** [comment]
@pradeepkchaturvedi is this still a pain point for you? I'm starting to think about 1.10 stories.

**Comment by reasonerjt** [comment]
@pradeepkchaturvedi  is this requirement for showing the progress or showing the result?

Do you wanna show the size of the PV or the snapshot?

I don't think we have bandwidth to handle this in v1.10 but it would be helpful if we can clarify the requirement.
